README.md

```diff
@@ -17,121 +23,13 @@ some may not be fully compatible as the version available in NixOS or
-      system = "x86_64-linux";
-      modules = [
-        home-manager.nixosModules.home-manager
-        ({
-          home-manager.users.exampleUser = lib.mkMerge [
-            nix-doom-emacs.hmModule
-            { ... }: {
-              programs.doom-emacs = {
-                enable = true;
-                doomPrivateDir = ./doom.d;
-              };
-            }
-          ];
-        })
-      ];
-    };
-  };
-}
-```
-
-## Under the hood
-
-This expression leverages
-[nix-straight.el](https://github.com/nix-community/nix-straight.el) under the hood for
-installing dependencies. The restrictions of that package apply here too.
-
-## Usage
-
-instead of running emacs.d/bin/doom, once you have update your config files (packages.el, init.el, config.el), rebuild doom-emacs with nix. If you are using home-manager, simply run `home-manager switch`
-
-## Troubleshooting
-
-On macOS on a fresh install, you might run into the error `Too many files open`. running `ulimit -S -n 2048` will only work for the duration of your shell and will fix the error
-
-## Installing emacs packages
-
-In the initial packages.el instructions for how to install packages can be
-found. However some packages might require a particular software dependency to
-be installed. Trying to install those would give you an error of the type:
-`Searching for program: No such file or directory, git` (Missing git dependency)
-Here is how you would go installing
-[magit-delta](https://github.com/dandavison/magit-delta) for example (which
-requires git).
-
-Under the line:
-`doomPrivateDir = ./doom.d;`
-in your configuration, you would add the following:
-
-```Nix
-{
-  emacsPackagesOverlay = self: super: {
-     magit-delta = super.magit-delta.overrideAttrs (esuper: {
-       buildInputs = esuper.buildInputs ++ [ pkgs.git ];
-     });
-  };
-}
-```
-
-To make the git dependency available. trying to rebuild doom-emacs with
-`home-manager switch` should work correctly now.
-
-## Using the daemon
+# Quick Start
 
-To use the daemon, simply enable the emacs service (with `NixOS`, `home-manager`
-or `nix-darwin`) and use the doom emacs package. `doom-emacs` will need to be
-referenced at the top of your config file.
+If you want to get a taste of nix-doom-emacs, you can run ``nix run github:nix-community/nix-doom-emacs``
+Which will run nix-doom-emacs with an example configuration. 
 
-```nix
-{
-  services.emacs = {
-    enable = true;
-    package = doom-emacs; # Not needed if you're using the Home-Manager module instead
-  };
-}
-```
+Pick which setup you're using here (if you're not using NixOS or Home-Manager, then you should use standalone):
 
-to connect to the daemon you can now run `emacsclient -c`
+| Home-Manager | NixOS | Standalone |
+|      ---     |  ---  |    ---     |
+| [Flake + Home-Manager](./docs/reference.md#home-manager) | [NixOS](./docs/reference.md#nixos) | [Standalone](./docs/reference.md#standalone) |
```

Comment:
Add a `## Hacking` section mentioning the license, the open issues, the PR wanted label, etc.. Also that contributions are welcome (:

Reply:
What do you mean by mentioning the open issues? Just linking them?

> # Hacking
> This project is licensed under MIT. Our issue tracker has some [open issues](https://github.com/nix-community/nix-doom-emacs/issues?q=is%3Aopen+is%3Aissue). The PR wanted label is for issues that need PRs to fix them. Contributions are welcome

This is what I have in mind, is it fine @ckiee?

Reply:
«under the MIT license», comma between «open issues» and «The PR wanted label». Better to commit so I can just do a nice-and-easy <code>\`\`\`suggestion</code>.